Tasks/Common/docker-common-v2/pipelineutils.ts

```diff
@@ -19,31 +19,25 @@ function addLabel(hostName: string, labelName: string, variableName: string, lab
     }
 }
 
-function addCommonLabels(hostName: string, labels: string[], addPipelineData?: boolean): void {
```

Comment:
I think we can keep this. This is about adding default labels or not. Sync-up with Vishal.

Reply:
Added back the check for pipeline metadata in docker task as it is used for adding labels.